src/Cartography.StaticMap.Abstractions/ZoomLevel.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Globalization;
 
-namespace Cartography.StaticMap;
+namespace Cartography;
```

Comment:
Didn't you say that ZoomLevel had a definition is more than one Cartography package? If so why remove the more precise namespace? If you have 2 types with the same name and namespace, you might not be able to use both package in the same project.

Reply:
Fixed.